Source/WebCore/platform/graphics/cocoa/SourceBufferParserWebM.cpp

```diff
@@ -1458,6 +1459,8 @@ void SourceBufferParserWebM::flushPendingAudioSamples()
         return;
     ASSERT(m_audioInfo);
     m_queuedAudioSamples.setInfo(m_audioInfo.copyRef());
+    m_queuedAudioSamples.setDiscontinuity(m_audioDiscontinuity);
```

Comment:
Is this ok if `m_audioDiscontinuity` is `nullopt`? Can it ever get to that point?